Update instructions for Ubuntu 20.04 (#1588)
anbox/anbox: docs/install.md

## Install necessary kernel modules

### On Ubuntu 20.04

> There is no need to install those modules on Ubuntu 20.04. However, you may get an error loading `ashmem_linux` if SecureBoot is enabled:

Reviewer: I'd do this
```suggestion
There is no need to install those modules on a recent Ubuntu or Debian: Even Ubuntu 18.04 LTS and Debian Stable have got the modules. However, you may get an error loading `ashmem_linux` if SecureBoot is enabled:
```

```
 $ sudo modprobe ashmem_linux
 modprobe: ERROR: could not insert 'ashmem_linux': Operation not permitted
```

You can confirm that SecureBoot is enabled by running the following command:

```
$ sudo mokutil --sb-state
SecureBoot enabled
```

There are two ways around this. One is to disable the SecureBoot: https://wiki.ubuntu.com/UEFI/SecureBoot/DKMS. 
Following [this post](https://github.com/anbox/anbox/issues/1570), the other way is to sign the `ashmem_linux` kernel module yourself. Note that you may have to enroll your own key, as described [here](https://ubuntu.com/blog/how-to-sign-things-for-secure-boot).

> ### Before Ubuntu 20.04

Reviewer: ```suggestion
### Before Ubuntu 19.04
```

> 

Reviewer: ```suggestion

You should have the modules on your system already. However, you can still try to use the ppa (no longer maintained) if `sudo modprobe ashmem_linux binder_linux` can't find the modules.

```
